contracts/libraries/RemainingInvalidatorLib.sol

```diff
@@ -66,7 +66,7 @@ library RemainingInvalidatorLib {
      */
     function remains(uint256 remainingMakingAmount, uint256 makingAmount) internal pure returns(RemainingInvalidator) {
         unchecked {
-            return RemainingInvalidator.wrap(remainingMakingAmount - makingAmount + 1);
+            return RemainingInvalidator.wrap(~(remainingMakingAmount - makingAmount));
```

Comment:
It seems like the documentation for `RemainingInvalidatorLib` library needs to be updated.

> When order is filled partially, the remaining invalidator is the remaining amount plus 1

Currently, it is a negation of the remaining amount.

Reply:
fixed